start.bash

```diff
@@ -156,14 +156,15 @@ expose default pod/sqlflow-server 8888:8888
 expose default pod/sqlflow-server 3306:3306
 expose default pod/sqlflow-server 50051:50051
 
-jupyter_addr=$(kubectl logs pod/sqlflow-server notebook | grep -o -E "http://127.0.0.1[^?]+\?token=.*" | head -1)
+# Get Jupyter Notebook's token, for single-user mode, we disabled the token checking
+# jupyter_addr=$(kubectl logs pod/sqlflow-server notebook | grep -o -E "http://127.0.0.1[^?]+\?token=.*" | head -1)
```

Comment:
The multi-users mode also doesn't need the token, so maybe we can remove these two lines.

Reply:
Will do it in next PR